mesonbuild/mesonmain.py

```diff
@@ -303,6 +303,12 @@ def run(args, mainfile=None):
             return mconf.run(remaining_args)
         elif cmd_name == 'wrap':
             return wraptool.run(remaining_args)
+        elif cmd_name == 'runpython':
+            import runpy
+            script_file = remaining_args[0]
+            sys.argv[1:] = remaining_args[1:]
+            runpy.run_path(script_file, run_name='__main__')
+            sys.exit(0)
```

Comment:
Does this mean that scripts that get run with the internal interpreter will always return 0 (ie: success)?

Reply:
Yes, unless they throw an exception or sys.exit(nonzero_number). Which is what they need to do with standalone scripts anyway.

Reply:
@jpakkane perfect. :+1: